lightning/src/offers/offer.rs

```diff
@@ -1527,7 +1518,7 @@ mod bech32_tests {
 
 	#[test]
 	fn fails_parsing_bech32_encoded_offer_with_invalid_tlv_data() {
-		let encoded_offer = "lno1qcp4256ypqpq86q2pucnq42ngssx2an9wfujqerp0y2pqun4wd68jtn00fkxzcnn9ehhyec6qgqsz83qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczlqsp9nyu4phcg6dqhlhzgxagfu7zh3d9re0sqp9ts2yfugvnnm9gxkcnnnkdpa084a6t520h5zhkxsdnghvpukvd43lastpwuh73k29qsyqqqqq";
+		let encoded_offer = "lno1pqps7sjqpgtyzm3qv4uxzmtsd3jjqer9wd3hy6tsw35k7msjzfpy7nz5yqcnygrfdej82um5wf5k2uckyypwa3eyt44h6txtxquqh7lz5djge4afgfjn7k4rgrkuag0jsd5xvxgqqqqq";
```

Comment:
Hmm can't find this one and the two above in spec. Maybe I'm missing them.

Reply:
Wait there's tons of them above. Nvm, probably just accidentally added a character when searching. I see them :)